packages/@kragle/ide/src/tools/node-bindings-editor/use-layout.tsx

```diff
@@ -81,6 +81,277 @@ function calculateLayout(document: SceneDocument): Layout {
   return { canvasWidth, canvasHeight, nodeBoxPositions };
 }
 
+export interface Layout2 {
+  readonly canvasWidth: number;
+  readonly canvasHeight: number;
+  readonly NodeBoxes: Map<string, [number, number]>;
+  /**
+   * Map<NodeID, [Xcoordinate, Ycoordinate, NodeID]>
+   */
+  readonly Paths: Map<
+    string,
+    [XCoordinate: number, YCoodrinate: number, NodeID: string]
+  >;
+}
+
+interface nodesBox {
+  readonly column: number;
+  row?: [number, number];
+  Xcoordinate?: number;
+  Ycoordinate?: number;
```

Comment:
In JavaScript/TypeScript ist die uebliche naming convention, Typ-Namen gross zu schreiben, und Variablen-Namen klein; also `NodesBox` und `xCoordinate`.

Reply:
Hast du mir schon öfter gesagt, ich werds noch öfter vergessen.

Kann das nicht `Prettier` machen?